src/clinical/clinical-service.ts

```diff
@@ -69,19 +68,21 @@ export type PaginationQuery = {
 	sort: string;
 };
 
-export type PaginatedClinicalQuery = ClinicalDonorEntityQuery & PaginationQuery;
+type ClinicalDataPaginatedQuery = ClinicalDonorEntityQuery & PaginationQuery;
+
+export type ClinicalDataDbQuery = ClinicalDataPaginatedQuery & {
```

Comment:
"Clinical Data Database Query" -> "Clinical Database Query" to avoid redundancy?